### DISCUSSION TITLE
Removing dracula color scheme

### DISCUSSION THREAD

**Discussion by SunsetTechuila** [nimsandu/spicetify-bloom#273] [General]
It looks very bad to me and I don't want to keep a bad scheme or edit it. Also never seen anyone use it
Can I just remove it?

![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/d7aa7691-19d3-416c-9bfe-2a38f9d49d84)
![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/ea878098-4dcb-4f17-b89c-b61741ba7320)

**Comment by nimsandu** [comment]
@SunsetTechuila I think removing it will be alright. It was just there when I started developing this based on fluent theme. But i never wanted someone to use it.
